hare/algorithm.go

```diff
@@ -56,7 +57,7 @@ type State struct {
 }
 
 type StateQuerier interface {
```

Comment:
why do you need both `StateQuerier` and `Rolacle` interfaces

Reply:
for example, ConsensusProcess holds both Rolacle and StateQuerier

Reply:
There are parts (such as newMsg) that need only the IsActive functionality, better have the interfaces as short as possible, also better for UT.
I will, however, remove holding both of them.